[Cache] Updates cache keys options per plan (#17668)
cloudflare/cloudflare-docs: src/content/docs/cache/how-to/cache-keys.mdx

## Availability

> Cache keys options availability dependes per plan type.

Reviewer: ```suggestion
Cache keys options availability varies according to your plan.
```
